addOns/ascanrules/src/main/java/org/zaproxy/zap/extension/ascanrules/CrossSiteScriptingScanRule.java

```diff
@@ -677,6 +678,81 @@ private boolean performImageTagAttack(HtmlContext context, HttpMessage msg, Stri
         return false;
     }
 
+    private boolean performAttributeAttack(HtmlContext context, HttpMessage msg, String param) {
+        String attackString = "accesskey=\"x\" onclick=\"alert('1')\" b";
```

Comment:
We deliberately use `alert(1)` in the other attacks as theres a chance the single quotes could get blocked / filtered etc. 
Most of the other 'main' attacks are in constants at the top as well which makes it a bit easier to see the all, although I admit this is not really that consistant ;)

Reply:
I thought this should not be a "main" attack. It should be carried out only if the "main" attack fails :) which is the "tag closing" attack